src/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ add_subdirectory(graph)
 # Bindings subdirectories
 if (Python3_Development_FOUND AND NOT SKIP_PYBIND11)
```

Comment:
I should nest this logic like is done in https://github.com/gazebosim/gz-transport/pull/565

Reply:
https://github.com/gazebosim/gz-math/pull/662/commits/dcf5151fb06f259938fbce970659118437d5bf98